lens/request/request.proto

```diff
@@ -10,7 +10,12 @@ message IndexRequest {
     string objectIdentifier = 2;
 }
 
-message SearchRequest {
+message SimpleSearchRequest {
     // repeated means there can be 0 or more "keywords"
     repeated string keywords = 1;
 }
+
+message AdvancedSearchRequest {
```

Comment:
protobuf supports compound and nested types, why not just make this a single endpoint, where if you provide extra options itll make a more advances search request?

Reply:
Hmm okay I'll have to read about that will let you know when this is done